### PR TITLE
Feature/updated docs and versions

### PR DESCRIPTION
**Pull request by lmwilkigov**
## Context

Versions weren't consistent and docs were broken

## Changes proposed in this pull request

I've propagated updated version through pyproject.toml, library and core api. I've also spruced up the docs with a gif of the UI in action.

## Guidance to review

Check out the status endpoint in core api and build the docs

## Things to check

- [ ] I have added any new ENV vars in all deployed environments
- [ ] I have tested any code added or changed
- [ ] I have run integration tests

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -4,7 +4,7 @@ target-version = 'py311'
 
 [tool.poetry]
 name = "redbox"
-version = "1.0.0"
+version = "0.0.3"
```
Comment (brunns) [inline pyproject.toml:7]: Aren't we on `0.3.0`?

```suggestion
version = "0.3.0"
```

File: redbox/__init__.py [hunk 2]
```diff
@@ -1,3 +1,3 @@
 """Redbox is a Python library for working with the Redbox API, data and services."""
 
-__version__ = "0.9.0"
+__version__ = "0.0.3"
```
Comment (brunns) [inline redbox/__init__.py:3]:
```suggestion
__version__ = "0.3.0"
```

**Review by gecBurton — APPROVED** [review]
LGTM

we should think about a GH action to ensure that the version number is correct on release

**Comment by brunns** [comment]
> LGTM
> 
> we should think about a GH action to ensure that the version number is correct on release

And also generate the changelog, assuming we think #638 is the right approach.

**Review by brunns — APPROVED** [review]
LGTM.

**Pull request merged**